conda-forge.yml

```
@@ -1,6 +1,9 @@
+conda_build:
+  pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
+provider:
+  linux_aarch64: default
```

```suggestion
  linux_aarch64: linux_64
```
TravisCI does not work - there are no CI checks for aarch64 at the moment. 
Let's try with cross compilation

---

Actually this is wrong. It should use `build_platform` instead

---

thanks - done.